javascript-forms/javascript-forms-quiz-notes.md

```diff
@@ -0,0 +1,53 @@
+# javascript-forms-quiz-notes
+
+## Quiz Questions
+
+After completing this exercise, you should be able to discuss or answer the following questions:
+
+- What event is fired when a user places their cursor in a form control?
+
+- What event is fired when a user's cursor leaves a form control?
+
+- What event is fired as a user changes the value of a form control?
+
+- What event is fired when a user clicks the `"submit"` button within a `<form>`?
+
+- What does the `event.preventDefault()` method do?
+
+- What does submitting a form without `event.preventDefault()` do?
+
+- What property of a form element object contains all of the form's controls.
+
+- What property of a form control object gets and sets its value?
+
+- What is one risk of writing a lot of code without checking to see if it works so far?
+
+- What is an advantage of having your console open when writing a JavaScript program?
```

Comment:
Don't forget the quiz questions!

Reply:
I realized I forgot them right after I submitted the pull request, my bad!